Sources/XMTP/ConversationV2.swift

```diff
@@ -15,11 +15,12 @@ public struct ConversationV2Container: Codable {
 	var conversationID: String?
 	var metadata: [String: String] = [:]
 	var peerAddress: String
+	var createdAt: UInt64?
 	var header: SealedInvitationHeaderV1
 
 	public func decode(with client: Client) -> ConversationV2 {
 		let context = InvitationV1.Context(conversationID: conversationID ?? "", metadata: metadata)
-		return ConversationV2(topic: topic, keyMaterial: keyMaterial, context: context, peerAddress: peerAddress, client: client, header: header)
+		return ConversationV2(topic: topic, keyMaterial: keyMaterial, context: context, peerAddress: peerAddress, client: client, createdAt: createdAt ?? 0, header: header)
```

Comment:
This allows createdAt to be nil so let it be nil.

```suggestion
		return ConversationV2(topic: topic, keyMaterial: keyMaterial, context: context, peerAddress: peerAddress, client: client, createdAt: createdAt, header: header)
```